source/OptimaJet.DWKit.StarterApplication/Forms/Groups/UpdateForm.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Linq;
+using JsonApiDotNetCore.Internal;
+using Microsoft.EntityFrameworkCore;
+using Optimajet.DWKit.StarterApplication.Models;
+using OptimaJet.DWKit.StarterApplication.Repositories;
+using OptimaJet.DWKit.StarterApplication.Services;
+
+namespace OptimaJet.DWKit.StarterApplication.Forms.Groups
+{
+    public class UpdateForm : BaseForm
+    {
+        public UserRepository UserRepository { get; set; }
+        public GroupRepository GroupRepository { get; set; }
+        public ICurrentUserContext CurrentUserContext { get; }
+        public UpdateForm(
+            UserRepository userRepository,
+            GroupRepository groupRepository,
+            ICurrentUserContext currentUserContext)
+        {
+            UserRepository = userRepository;
+            GroupRepository = groupRepository;
+            CurrentUserContext = currentUserContext;
+         }
+        public bool IsValid(int id, Group group)
+        {
+            var retVal = true;
+            //If changing organization, validate the change
+            if (group.OwnerId > 0)
+            {
+                var currentUser = UserRepository.GetByAuth0Id(CurrentUserContext.Auth0Id).Result;
```

Comment:
the current user should already exist in the CurrentUserContext, no need to trigger another query

Reply:
Current user is not accessible in the current user context.  And while CurrentUser is available at the Controller level, it is not easily accessible at the service level where the forms are called.

Reply:
You could pass it down to the service? maybe? idk